<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="TW">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>481</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>add my drawing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="take_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>take drawing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>del drawing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
